cppseminar/testservice/Controllers/TestRunsController.cs

```diff
@@ -166,5 +166,56 @@ public async Task<ActionResult> OnPostAsync([FromBody] TestRun testRun)
 
         return Ok();
     }
+    //counts number of used tests
+    [HttpGet("count/{userEmail}/{testId}")]
+    public async Task<ActionResult<long>> CountTestRuns([FromRoute] string userEmail, [FromRoute] string testId)
+    {
+        System.Console.WriteLine("Tu som v test run controller");
+        _logger.LogTrace("Retrieving concrete test {test} {user}", testId, userEmail);
+
+        long CountedRuns;
 
+        try
+        {
+            CountedRuns = await _testRuns.GetCountAsync(userEmail, testId);
+        }
+        catch (FormatException e)
+        {
+            _logger.LogWarning("Wrong input format, userEmail {userEmail} testId {testId}. {e}", userEmail, testId, e);
+            return StatusCode(400);
+        }
+        catch (Exception e)
+        {
+            _logger.LogError("Error retrieving data for run id {test} {e}", testId, e);
+            return StatusCode(500);
+        }
+
+        return CountedRuns;
+    }
+
+    [HttpPost("setCounted/{testRunId}")]
```

Comment:
I think setCounted is wrong here, it should be just count. @lukas-manduch you know http more than me, what do you think?